person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    is_of_age? || @parent_permission
```

Comment:
- [ ] Kindly fix the name for your age method which you define it `of_age?` and when you called it `is_of_age?`

Reply:
Thank you. I will sort it right away